event_routing_backends/processors/mixins/base_transformer.py

```diff
@@ -44,7 +44,7 @@ def find_nested(source_dict, key):
         Returns:
             ANY
         """
-        def _find_nested(event_dict):
+        def _find_nested(event_dict, default=None):
```

Comment:
Why are you adding that extra variable called default??? Is necessary added if seems the none is working at the same way

Reply:
ok maybe for calling the functions seems easier to read but it could be default_value. But is a good helper